api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -117,5 +117,31 @@
                                      baseTableName="patient_program_attribute" baseColumnNames="voided_by"
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
-    
+
+	<changeSet id="20181402-TRUNK-5339" author="esirkings">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="encounter_diagnosis" />
+			</not>
+		</preConditions>
+		
+		<comment>Creating encounter_diagnosis table</comment>
+		
+		<createTable tableName="encounter_diagnosis">
+			<column name="diagnosis_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false" />
+			</column>
+			<column name="encounter_id" type="int"><constraints nullable="false" /></column>
+			<column name="condition_id" type="int"><constraints nullable="false" /></column>
+			<column name="certainty" type="varchar(255)"><constraints nullable="false" /></column>
+			<column name="rank" type="int"><constraints nullable="false" /></column>
+		</createTable>
+		
+		<addForeignKeyConstraint constraintName="encounter_id_fk"
```

Comment:
Make the fk names unique by appending the table name. e.g encounter_diagnosis_encounter_id_fk